rm/rm-client/src/main/java/org/ow2/proactive/resourcemanager/core/properties/PAResourceManagerProperties.java

```diff
@@ -323,7 +323,32 @@
     /**
      * Defines the maximum number of RMEvents which can be sent to the client in one request.
      */
-    RM_REST_MONITORING_MAXIMUM_CHUNK_SIZE("pa.rm.rest.monitoring.maximum.chunk.size", PropertyType.INTEGER, "100");
+    RM_REST_MONITORING_MAXIMUM_CHUNK_SIZE("pa.rm.rest.monitoring.maximum.chunk.size", PropertyType.INTEGER, "100"),
+
+    /**
+     * Enables or disables asynchronous logging for server
+     */
+    LOG4J_ASYNC_APPENDER_ENABLED("pa.log4j.async.appender.enabled", PropertyType.BOOLEAN, "true"),
+
+    /**
+     * When asynchronous logging is enabled, also cache file appenders inside a cache structure (leaves file descriptors open while a job or task is running)
+     */
+    LOG4J_ASYNC_APPENDER_CACHE_ENABLED("pa.log4j.async.appender.cache.enabled", PropertyType.BOOLEAN, "false"),
+
+    /**
+     * Defines the buffer size used in asynchronous appenders
+     */
+    LOG4J_ASYNC_APPENDER_BUFFER_SIZE("pa.log4j.async.appender.buffer.size", PropertyType.INTEGER, "10000"),
+
+    /**
+     * Defines the size of the thread pool of the AsynchFileAppender
+     */
+    LOG4J_ASYNC_APPENDER_POOL_SIZE("pa.log4j.async.appender.pool.size", PropertyType.INTEGER, "1"),
+
+    /**
+     * Defines the AsynchFileAppender flush timeout
+     */
+    LOG4J_ASYNC_APPENDER_FLUSH_TIMOUT("pa.log4j.async.appender.flush.timeout", PropertyType.INTEGER, "300");
```

Comment:
50ms seems better

Reply:
how come 50 is in any sense better than 300?